Dockerfile

```diff
@@ -0,0 +1,41 @@
+# base image
+FROM python:3.10
```

Comment:
pushed image on githubio with appropriate tag

Reply:
Alternatively we could also do a full build of batteries included docker file... with openms + tools + pyOpenMS +thirdparty.
Maybe that would make it easier to deal with experimental branches.